API.md

```diff
@@ -238,20 +240,22 @@ Running app guarantees to have at least one open window.
 Handlers registered via [App.serveHandler](#appservehandlerhandler) and [Window.serveHandler](#windowservehandlerhandler) receive parameter of this upon every network request.
 
 #### HttpRequest.abort()
+- `return`: <[Promise]>
 
 Aborts request.
 
 #### HttpRequest.continue()
 
 Proceeds with the default behavior for this request. Either serves it from the filesystem or defers to the browser.
 
-#### HttpRequest.fulfill(options)
-- `options`: <[Promise]<[Object]>>
-  - `status` <[number]> HTTP status code (200, 304, etc), optional, defaults to 200.
-  - `headers` <[Object]> HTTP response headers
+#### HttpRequest.fulfill([options])
```

Comment:
this object is required...

```suggestion
#### HttpRequest.fulfill(options)
```

Reply:
Sorry, fixed.

Reply:
Should I squash and edit first commit message?